protocol/ignored-changes.xml

```diff
@@ -25,4 +25,15 @@
     <!-- final modifier was added to class -->
     <differenceType>3003</differenceType>
   </difference>
+  <!-- same intents are renamed for version 1.0.0 -->
+  <difference>
+    <className>io/zeebe/protocol/record/intent/MessageIntent</className>
+    <differenceType>6001</differenceType>
+    <field>DELETE</field>
+  </difference>
+  <difference>
+    <className>io/zeebe/protocol/record/intent/MessageIntent</className>
+    <differenceType>6001</differenceType>
+    <field>DELETED</field>
+  </difference>
```

Comment:
Maybe we just remove the clirr for now :sweat_smile:

Reply:
I saw that @menski did it on his branch already

Reply:
Interesting point. I'll keep it for the PR because I like that it highlights the breaking changes. But I'm open to disable it until the release.